main.go

```diff
@@ -1,14 +1,14 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"github.com/drone/drone-plugin-go/plugin"
+	try "gopkg.in/matryer/try.v1"
 	"net/http"
```

Comment:
go convention is group imports, starting with std library, and then external packages.
https://github.com/golang/go/wiki/CodeReviewComments#imports